hadoop-ozone/integration-test/src/test/java/org/apache/hadoop/ozone/om/TestOMDbCheckpointServletInodeBasedXfer.java

```diff
@@ -308,6 +324,62 @@ public void testSnapshotDBConsistency() throws Exception {
     assertNotNull(value);
   }
 
+  @ParameterizedTest
+  @ValueSource(booleans = {true, false})
+  public void testWriteDBToArchive(boolean isSstFile) throws Exception {
```

Comment:
suggest to replace isSstFile with a more understandable name, like expectOnlySstFiles